src/protocols/secure_channel/CASESession.h

```diff
@@ -269,11 +262,8 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
 
     Crypto::Hash_SHA256_stream mCommissioningHash;
     Crypto::P256PublicKey mRemotePubKey;
-#ifdef ENABLE_HSM_CASE_EPHEMERAL_KEY
-    Crypto::P256KeypairHSM mEphemeralKey;
-#else
-    Crypto::P256Keypair mEphemeralKey;
-#endif
+    Crypto::P256Keypair * mEphemeralKey = nullptr;
+    Crypto::P256Keypair mEphemeralKey2;
```

Comment:
What is this used for?

Reply:
Good catch, that was a side effect of testing a test